src/ONYXKEYS.ts

```diff
@@ -326,6 +326,8 @@ const ONYXKEYS = {
         ADD_DEBIT_CARD_FORM: 'addDebitCardForm',
         ADD_DEBIT_CARD_FORM_DRAFT: 'addDebitCardFormDraft',
         WORKSPACE_SETTINGS_FORM: 'workspaceSettingsForm',
+        WORKSPACE_CATEGORY_CREATE_FORM: 'workspaceCategoryCreate',
+        WORKSPACE_CATEGORY_CREATE_FORM_DRAFT: 'workspaceCategoryCreateDraft',
```

Comment:
Is it necessary to create a draft form key even we don't have draft option?

Reply:
Yea, I found that weird too, but type checks fail without it.

Reply:
Cool.